ansible/all-in-one/roles/xsnippet/templates/xsnippet-webserver-spa.conf.j2

```diff
@@ -38,8 +36,8 @@ server {
     gzip_types text/plain text/css application/json application/x-javascript text/javascript application/javascript image/svg+xml;
     gzip_min_length 1024;
 
-    location ~ ^/(\d+)/raw/?$ {
-        proxy_pass http://xsnippet_web_backend/snippets/$1/raw;
+    location ~ ^/([a-zA-Z0-9]+)/raw/?$ {
+        js_content http.get_raw_snippet;
```

Comment:
nit: I'm not sure `http.js` is a good name for xsnippet-web's companion routes. I mean, it's a big generic.

Reply:
Sure! I'll rename to xsnippet_api.js to denote the fact that it contains helper functions to make requests to the API.